app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,12 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+  end
 
-  def show; end
+  def show
+    @post = Post.find(params[:id])
+    @user = @post.user
+    @comments = @post.comments
```

Comment:
- Please fix the error as seen below to go to the next project
![Screenshot from 2022-08-18 14-11-35](https://user-images.githubusercontent.com/44624138/185405346-d5be56d8-aef3-482b-8ad9-072089686369.png)

Reply:
@rotimiazeez, thanks for the review. However, I think there's an Issue on your end. The code works perfectly on my side. If you could please get on a call with me, it'd be a great help.

Reply:
@Dinma-Faith that will be great

Reply:
@Dinma-Faith My sincere apology for the mix-up, I believe it was an error from my end and I'm looking into it. Kindly submit the project again and it will be approved 🙏🏽 .

Reply:
@rotimiazeez , thanks for your time